src/shared/ui/toast.tsx

```diff
@@ -0,0 +1,45 @@
+import error from '@/src/assets/icon/medium/error.svg'
+import logo from '@/src/assets/images/(logo)/logo_white.svg'
+import Image from 'next/image'
+
+type ToastProps = {
+  type: 'warning' | 'notice'
+  message: string
+}
+
+const TOAST_STYLES = {
+  warning:
+    'fixed top-[30px] left-1/2 transform -translate-x-1/2 z-[10001] flex flex-row items-center justify-center w-[355px] h-[55px] px-[20px] py-[10px] bg-white rounded-[10px] shadow-custom border-[1.5px] border-error',
```

Comment:
```suggestion
    'fixed top-[30px] left-1/2 transform -translate-x-1/2 z-[10001] flex flex-row items-center justify-center w-[355px] h-[65px] px-[20px] py-[10px] bg-white rounded-[10px] shadow-custom border-[1.5px] border-error',
```
디자인 확인해보니 모든 양식에서 h-[65px] 입니다!